testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -220,8 +220,13 @@ def test_molecule_tag(self, LAMMPSDATAWriter_molecule_tag):
                      err_msg="resids different after writing",)
 
 
-def test_datawriter_universe(tmpdir):
-    fn = str(tmpdir.join('out.data'))
+@pytest.mark.parametrize('filename', ['out.data', 'out.data.bz2'])
```

Comment:
```suggestion
@pytest.mark.parametrize('filename', ['out.data', 'out.data.bz2', 'out.data.gz'])
```
?

Reply:
Oops forgot the one I was actually meant to fix 🙃